src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.ts

```diff
@@ -34,20 +32,6 @@ export class EditTechRecordButtonComponent implements OnDestroy {
     private routerService: RouterService
   ) {}
 
-  // ngOnInit() {
-  //TODO: V3 is this needed? causing issues when updating record feels like the logic should be handled in the parent
```

Comment:
Can we get rid of this commented out code or does it need to be uncommented and fixed?